tofu/main/aws/terraform.tf

```
@@ -1,9 +1,9 @@
 terraform {
-  required_version = "1.8.2"
+  required_version = ">= 1.8.1"
```

```suggestion
  required_version = "1.8.2"
```

I prefer sticking to the vendored version, which is now 1.8.2